release/mixer.sh

```diff
@@ -96,7 +96,14 @@ generate_bump() {
 
     # Remove bundles pending deletion
     section "Bundle Deletion"
-    for i in $(grep -lir "\\[STATUS\\]: Pending-Delete" upstream-bundles/ local-bundles/); do
+    local bundle_folders
+    if "${IS_UPSTREAM}"; then
```

Comment:
Won't block the merge, but lets have the usage/meaning of those new input for this script to be documented on the top of this file.